### PR TITLE
Hook up new team wizard

### PR DESCRIPTION
**Pull request by jakob223**
cc @keybase/y2ksquad

### CODE REVIEW

File: shared/teams/add-members-wizard/add-from-where.tsx [hunk 1]
```diff
@@ -3,20 +3,39 @@ import * as Kb from '../../common-adapters'
...
+  }
+}
+
 const AddFromWhere = () => {
   const dispatch = Container.useDispatch()
   const nav = Container.useSafeNavigation()
 
   const teamID = Container.useSelector(s => s.teams.addMembersWizard.teamID)
   const newTeam: boolean = teamID === Types.newTeamWizardTeamID
+  const error = Container.useSelector(s => (newTeam ? s.teams.newTeamWizard.error : undefined))
```
Comment (buoyad) [inline shared/teams/add-members-wizard/add-from-where.tsx:35]: This is if you hit "skip" and creating the team fails?

File: go/service/teams.go [hunk 2]
```diff
@@ -60,6 +62,83 @@ func (h *TeamsHandler) TeamCreate(ctx context.Context, arg keybase1.TeamCreateAr
...
+			SendChatNotification: false,
+			EmailInviteMessage:   teamInfo.EmailInviteMessage,
+		}
+
+		unaddedUsers, err := h.TeamAddMembersMultiRole(ctx, arg4)
+		if err != nil {
+			errs = append(errs, err)
+		}
+		if unaddedUsers.NotAdded != nil {
+			errs = append(errs, fmt.Errorf("could not add members to team: %v", unaddedUsers.NotAdded))
```
Comment (buoyad) [inline go/service/teams.go:128]: I think you need a length check. This keeps happening with an empty array for me.

File: shared/actions/teams.tsx [hunk 3]
```diff
@@ -1505,6 +1505,34 @@ const startAddMembersWizard = (_: TeamsGen.StartAddMembersWizardPayload) =>
...
+    users: state.teams.addMembersWizard.addingMembers.map(member => ({
+      assertion: member.assertion,
+      role: RPCTypes.TeamRole[member.role],
+    })),
+  }
+  try {
+    const teamID = await RPCTypes.teamsTeamCreateFancyRpcPromise({teamInfo}, Constants.teamCreationWaitingKey)
+    return [
+      TeamsGen.createFinishedNewTeamWizard(),
+      TeamsGen.createFinishedAddMembersWizard(),
```
Comment (buoyad) [inline shared/actions/teams.tsx:1529]: prefer to remove this and have the store changes / nav be a side effect of `finishedNewTeamWizard`

File: shared/teams/add-members-wizard/add-from-where.tsx (hunk 1 above)
Comment (jakob223) [reply shared/teams/add-members-wizard/add-from-where.tsx:35]: yeah. Should I rename to `createTeamError`?

Comment (buoyad) [reply shared/teams/add-members-wizard/add-from-where.tsx]: sounds good

**Pull request merged**